tests/devices/default_qubit/test_default_qubit.py

```diff
@@ -1879,7 +1875,7 @@ def circ_expected():
 
     @pytest.mark.parametrize(
         "mp, expected_shape",
-        [(qml.sample(wires=[0]), (5,)), (qml.classical_shadow(wires=[0]), (2, 5, 1))],
```

Comment:
Don't we still want to test the single wire case?

Reply:
I don't actually remember why I changed this. Added the single wire test cases back, and also kept the multi-wire tests.